[User-Proc 4.1d] // Экспорт данных произвел GSBIS 04/03/2015 15:27:24 БД "-db /home2/bis/quit41d/db/bisquit,-S bq41d,-N TCP,-ld bisquit,-U qbis,-P,-n 300"
0 ? "ОТЧЕТЫ ГЕНЕРАТОРА" "Base" 1 "RepGen" 2968 yes "Корневой раздел генератора отчетов" ""
0 "izvest" "Уведомление по суммам транз.счетов" "base" 2968 "RepGen" 149119 yes "" ""
0 "izvestnr" "Извещение: Средства от нерезидента" "base" 2968 "RepGen" 149120 yes "" ""
.
[Reports] // Экспорт данных произвел GSBIS 04/03/2015 15:27:24 БД "-db /home2/bis/quit41d/db/bisquit,-S bq41d,-N TCP,-ld bisquit,-U qbis,-P,-n 300"
"izvest" 1 "<!REPEAT[izvest],BEGIN!>"
"izvest" 2 "<!PAGE!>"
"izvest" 3 "УВЕДОМЛЕНИЕ № <#docnum[izvest]#>"
"izvest" 4 "о поступлении валютных средств <#date1[izvest]#>."
"izvest" 5 ""
"izvest" 6 "Наименование уполномоченного           Омский филиал ОАО ""Плюс Банк"""
"izvest" 7 "банка:                                 Отдел валютного контроля "
"izvest" 8 "                                       и документарных операций"
"izvest" 10 "Почтовый адрес:	           644024, Россия, г. Омск, Газетный пер., 6"
"izvest" 12 "Наименование организации:  <#name_klnt[izvest]#>"
"izvest" 14 "Почтовый адрес: <#adr_klnt[izvest]#>"
"izvest" 16 "В пользу Вашего предприятия поступили средства в сумме:"
"izvest" 18 "Код валюты: <#currency[izvest]#> Сумма цифрами <#amt[izvest]#>"
"izvest" 19 "Сумма прописью: <#amtl[izvest]#>,"
"izvest" 21 "которые <#date1[izvest]#> зачислены на Ваш транзитный счет <#acct[izvest]#>."
"izvest" 23 "Просим не позднее чем через 15 рабочих дней после даты зачисления средств "
"izvest" 24 "представить в наш банк справку о содержании валютной операции."
"izvest" 26 "Последняя дата предоставления справки о валютных операциях <#date2[izvest]#> г."
"izvest" 28 "При непредставлении Вами справки о содержании валютной операции денежные "
"izvest" 29 "средства с Вашего транзитного валютного счета списаны не будут."
"izvest" 31 "Телефон отдела валютного контроля                 (3812) 20-14-46"
"izvest" 32 "и документарных операций:"
"izvest" 35 ""
"izvest" 36 "	"
"izvest" 37 "_____________________ Ф.И.О., должность"
"izvest" 40 "               Отметка клиента о получении извещения:"
"izvest" 42 "<!REPEAT[izvest],END!> "
"izvestnr" 1 "<!REPEAT[izvest],BEGIN!>"
"izvestnr" 2 "<!PAGE!>"
"izvestnr" 3 "Отдел валютного контроля          Омский филиал ОАО ""Плюс Банк"""
"izvestnr" 4 "и документарных операций"
"izvestnr" 6 "ИЗВЕЩЕНИЕ N <#docnum[izvest]#>"
"izvestnr" 7 "О ПОСТУПЛЕНИИ ВАЛЮТЫ РФ"
"izvestnr" 9 "Наименование организации: <#name_klnt[izvest],70,L,1#>                                                                                  "
"izvestnr" 10 "<#name_klnt[izvest],70,L,*#>                                                                            "
"izvestnr" 11 "Почтовый адрес: <#adr_klnt[izvest]#> "
"izvestnr" 13 "В пользу Вашего предприятия поступили средства в сумме:"
"izvestnr" 14 "Код валюты: RUR Сумма цифрами: <#amt[izvest]#>"
"izvestnr" 15 "Сумма прописью: <#amtl[izvest]#>"
"izvestnr" 18 "N референса <#docnum[izvest]#>"
"izvestnr" 20 "Которые <#date1[izvest]#> зачислены на Ваш расчетный счет N "
"izvestnr" 21 "<#acct[izvest]#>"
"izvestnr" 22 "Просим не позднее  15-ти рабочих дней, начиная с даты, "
"izvestnr" 23 "следующей за датой зачисления средств, представить в наш "
"izvestnr" 24 "банк информацию об идентификации по контрактам поступивших  "
"izvestnr" 25 "денежных средств и  справку о валютных операциях."
"izvestnr" 27 "При непредоставлении информации об идентификации поступивших "
"izvestnr" 28 "денежных средств, рублевая выручка не будет учтена  как "
"izvestnr" 29 "поступившая по контракту, и данная информация будет "
"izvestnr" 30 "представлена в органы валютного контроля в соответствии с "
"izvestnr" 31 "п.9 ст.23 Федерального закона ""О валютном регулировании и "
"izvestnr" 32 "валютном контроле"" №173-ФЗ от 10.12.2003 г."
"izvestnr" 34 "Телефон отдела валютного контроля и документарных операций: "
"izvestnr" 35 "20-14-46/32-11-85"
"izvestnr" 39 "Отметка об отправке извещения:"
"izvestnr" 41 "<!REPEAT[izvest],END!> "
.
[Report-Fields] // Экспорт данных произвел GSBIS 04/03/2015 15:27:24 БД "-db /home2/bis/quit41d/db/bisquit,-S bq41d,-N TCP,-ld bisquit,-U qbis,-P,-n 300"
.
[Signs] // Экспорт данных произвел GSBIS 04/03/2015 15:27:24 БД "-db /home2/bis/quit41d/db/bisquit,-S bq41d,-N TCP,-ld bisquit,-U qbis,-P,-n 300"
"ВидДок" "" "149119" "user-proc" "izvest" ? ?
"ПчтРеж" "" "149119" "user-proc" "txt" ? ?
"ВидДок" "" "149120" "user-proc" "izvestnr" ? ?
"ПчтРеж" "" "149120" "user-proc" "txt" ? ?
.
